### DISCUSSION TITLE
How to intentionally wipe inventory

### DISCUSSION THREAD

**Discussion by Levardio** [Pryaxis/TShock#2089] [Q&A] [answered]
Hello, please, help, i wanted to host a small tournament, but some guys brought overpowered explosives, i backed up world, but i cant ban all explosives so this wont damage gameplay. So i decided to wipe memebers inventory at the start so they wouldnt bring unwanted stuff, but i dont know how.

**Comment by Levardio** [comment]
Or, at least, is there a way to check what items player have?

**Comment by Quinci135** [comment, marked as answer]
To control what items players have, you should use SSC (Server sided characters)
https://tshock.readme.io/docs/server-side-character-config

To see and edit what items a player has you could use the invsee plugin (anzhelikaO's updated fork)
https://github.com/AnzhelikaO/InvSee
Invsee/bin/Invsee.dll
